File: mobile/lib/domain/entities/event_entity.dart
// lib/domain/entities/event_entity.dart
class EventEntity {
  final String id;
  final String title;
  final String type;
  final DateTime startsAt;
  final DateTime? endsAt;
  final String timezone;
  final String location;
  final double fee;
  final String notes;
  final String? bandId;

  const EventEntity({
    required this.id,
    required this.title,
    required this.type,
    required this.startsAt,
    this.endsAt,
    required this.timezone,
    required this.location,
    required this.fee,
    this.notes = '',
    this.bandId,
  });
}

File: backend/src/modules/events/entities/event.entity.ts
import { EventStatus } from '@prisma/client';

export class Event {
  id!: string;
  title!: string;
  startsAt!: Date;
  endsAt?: Date | null;
  timezone!: string;
  type?: string | null;
  fee?: number | null;
  location!: string;
  description?: string | null;
  createdById!: string;
  createdAt!: Date;
  bandId!: string;
  updatedAt!: Date;
  status!: EventStatus;
}
